### PR TITLE
#184769261 Remove passwords in response's body

### PR DESCRIPTION
**Pull request by wilbrord2**
#### What does this PR do?
This PR is for excluding User-password and User-email in response body

#### Description of Task to be completed?
- In response body user password and email should be excluded

#### How should this be manually tested?
- Clone this repo git clone https://github.com/atlp-rwanda/e-comm-team-amigos-bn
- Switch your working directory cd e-comm-team-amigos-bn
- Create an .env file and update it with .env-example
- Install all the dependencies by running npm install
- Then run the project by typing npm start

#### Endpoints to be tested:
- ` /user/profile`
-  `/users`

#### Any background context you want to provide?
- Have Node Js installed in your local machine

#### What are the relevant pivotal tracker/Trello stories?

Story ID: [#184769261](https://www.pivotaltracker.com/story/show/184769261)

#### All Users Endpoint 
Screenshot (148)
Screenshot (154)

#### User Profile Endpoint

Screenshot (151)

Screenshot (155)

#### Tests screenshot 

Screenshot (153)

### CODE REVIEW

File: src/controllers/roles.js [hunk 1]
```diff
@@ -1,225 +1,247 @@
...
-      count: users.length,
-      response: users,
-    });
-  } catch (error) {
-    return res.status(500).json({ error: error.message });
-  }
-};
+    try {
+        const users = await models.User.findAll({
+            attributes: { exclude: ['password', 'email'] },
```
Comment (khevin2) [inline src/controllers/roles.js]: Was it necessary to remove the email also?

Comment (wilbrord2) [reply src/controllers/roles.js]: yes for user security purpose

**Review by GasanaJr — APPROVED** [review]
Good job

**Review by Emile-SHUMBUSHO — APPROVED** [review]

**Review by khevin2 — APPROVED** [review]

**Review by Habinezajanvier — APPROVED** [review]
LGTM

**Review by nshutijonathan — COMMENTED** [review]
LGTM

**Pull request merged**